substrate/frame/revive/uapi/src/host.rs

```diff
@@ -388,6 +388,9 @@ pub trait HostFn: private::Sealed {
 	/// - `offset`: Byte offset into the returned data
 	fn return_data_copy(output: &mut &mut [u8], offset: u32);
 
+	/// Returns the amount of ref_time left.
+	fn ref_time_left() -> u64;
```

Comment:
I am thinking, it would be nice to get links to https://www.evm.codes/?fork=cancun#37gas for all these evm apis

Reply:
Yeah why not, I will make a followup up for it :)